src/pages/iou/request/step/IOURequestStepCategory.tsx

```diff
@@ -78,7 +86,7 @@ function IOURequestStepCategory({
     const isSplitBill = iouType === CONST.IOU.TYPE.SPLIT;
     const canEditSplitBill = isSplitBill && reportAction && session?.accountID === reportAction.actorAccountID && TransactionUtils.areRequiredFieldsEmpty(transaction);
     // eslint-disable-next-line rulesdir/no-negated-variables
```

Comment:
do we still need to disable this eslint rule?

Reply:
I saw that but then isn't it needed for `const shouldShowEmptyState = !isLoading && !shouldShowCategory;` ?